comp/core/gui/guiimpl/platform_darwin.go

```diff
@@ -9,10 +9,27 @@ import (
 	"fmt"
 )
 
+const docURL = "https://docs.datadoghq.com/agent/basic_agent_usage/osx"
+
 func restartEnabled() bool {
 	return false
 }
 
 func restart() error {
 	return fmt.Errorf("restarting the agent is not implemented on non-windows platforms")
 }
+
+func logginInstructions() string {
+	return fmt.Sprintf(`<h3>Instructions</h3>
+<p>Please ensure you access the Datadog Agent Manager as follows:</p>
+<ul>
+	<li>- Through the systray app, by clicking on <strong>Open Web Ui</strong></li>
+	<li>- By running the following bash command: "<code>datadog-agent launch-gui</code>"</li>
+</ul>
+
+<p>For more information, please visit: <u><a href="%s">%s</a></u></p>
+
+<h4>Be Aware of Token Expiration</h4>
+The Datadog Agent parameter <code>GUI_session_expiration</code> (set in <code>datadog.yaml</code>) allows you to define a time expiration for the Datadog Agent Manager sessions.`,
```

Comment:
Suggestion (feel free to tweak accordingly for other platforms):
```
## Refreshing the Session

Please ensure you access the Agent GUI with one of the following:
- through the Agent's menu bar extras icon by clicking on "Open Web UI".
- by running the following bash command: "datadog-agent launch-gui"
For more information, please visit: <docurl>

Note: If you would like to adjust the GUI session timeout, you can modify the
`GUI_session_expiration` parameter in `datadog.yaml`.
```

Reply:
I updated the text with the main phrasing of your suggestion but kept `Datadog Agent Manager`  because it's the term used in all public documentation.
You can take a look at the updated version here: 8ac186ced8a9eb43ab035923ff43cfadc11ca57f